### ISSUE TITLE
when I press f4, I have both a new menu and an old one

### ISSUE THREAD

**Issue by Ne0Kit** [IsaiahPapa/blur-f4menu#1]
when I press f4, I have both a new menu and an old one, the old one closes and the new one does not

**Comment by IsaiahPapa** [comment]
Not going to lie, I have not played gmod in ages, and have not touched lua in the same about of time. I don't know how much the APIs have changed, and don't see myself to coding anything gmod related as I have a S&Box key. 

TLDR: This is old code, and probably broken. :( sorry

**Comment by thenamesjay** [comment]
go to darkrpmodification, and disableddefaults.lua and set f4 to true.

**Comment by thenamesjay** [comment]
> Not going to lie, I have not played gmod in ages, and have not touched lua in the same about of time. I don't know how much the APIs have changed, and don't see myself to coding anything gmod related as I have a S&Box key.
> 
> TLDR: This is old code, and probably broken. :( sorry

My brother in christ, I just downloaded this for the latest version of darkrp...
got lots of errors relating to it not being able to find a config file. I'm like hmm.. why is the config file even in the autorun, it should be with the other lua files in vgui.

guess what? It was in the wrong place. Everything works fine if you just move that single file to the right spot.
